ds701_book/21-Networks-I.qmd

```diff
@@ -818,8 +954,22 @@ plt.bar(degrees+0.5, degree_freq, 0.425, label = 'Actual')
 plt.xlabel('Degree', size = 14)
 plt.ylabel('Number of Nodes', size = 14)
 plt.legend(loc = 'best', fontsize = 14)
-plt.title('Degree Distribution of Football Network\nCompared to Random', size = 16);
+plt.title('Degree Distribution of Football Network\nCompared to Random', size = 16)
+plt.show()
 ```
 
 We can see the evidence of scheduling of games in this distribution: a much larger number of teams plays 11 games than would occur by chance.
 
+
+## Recap
```

Comment:
Could be fun to apply this network analysis to the NBA dataset. Just a comment.

Reply:
I agree, but I don't have the time for this.